### PR TITLE
Update UrlDispatcher Guide

### PR DESCRIPTION
**Pull request by cibersheep**
Correction of erroneous concept

### CODE REVIEW

File: appdev/guides/importing-CH-urldispatcher.rst [hunk 1]
```diff
@@ -89,21 +89,10 @@ And how do we manage the received URL?
 
 After the ``URLdispatcher`` sends the link to the correspondent app, we need to handle that URL or URI in the targeted app. Let’s see how to do that:
 
-In the main qml file, we need to add some code to know what to do with the dispatched URL. First add an `Arguments element <https://gitlab.com/ubports-linphone/linphone-simple/blob/master/qml/Main.qml#L189>`_ that holds the URL, as is done, for example, in the `Linphone app <https://open-store.io/app/linphone.cibersheep>`_. Also, we add `connection to the URI Handler <https://gitlab.com/ubports-linphone/linphone-simple/blob/master/qml/Main.qml#L200>`_ with a Connection element with ``UriHandler`` as a target.
+In the main qml file, we need to add some code to know what to do with the dispatched URL. Let's add `connection to the URI Handler <https://gitlab.com/ubports-linphone/linphone-simple/blob/master/qml/Main.qml#L200>`_ with a Connection element with ``UriHandler`` as a target.
```
Comment (doniks) [inline appdev/guides/importing-CH-urldispatcher.rst:92]: Let's add **a** connection

Comment (doniks) [reply appdev/guides/importing-CH-urldispatcher.rst:92]: I think it was nice to say explicitly that we are looking at one particular app (Linphone) as an example.

**Comment by cibersheep** [comment]
> Please don't include .pot changes. They get handled in another way (I think by the regular weblate merges). And it is muuuuuch easier to review PR's without .pot

How should I proceed? Close this and do a new MR? Leave it as is? Other?

File: appdev/guides/importing-CH-urldispatcher.rst (hunk 1 above)
Comment (cibersheep) [reply appdev/guides/importing-CH-urldispatcher.rst:92]: - Thanks. I'll edit that.
- About Linphone, true... I wasn't sure how to proceed as the sample code should change... but first I wanted to reflect it in the docs...

Comment (cibersheep) [reply appdev/guides/importing-CH-urldispatcher.rst]: Updated :)

**Comment by peter-pycom** [comment]
Either revert the change to the pot in this PR or make a new one. Whichever is easier for you

**Comment by cibersheep** [comment]
That should be it: 
- changed the rst text
- reverted the changes to the pot file

**Pull request merged**